metriken-exposition/src/parquet.rs

```diff
@@ -75,11 +75,12 @@ impl Default for ParquetOptions {
 /// tracks as summary statistics for every histogram encountered.
 #[derive(Default)]
 pub struct ParquetSchema {
-    counters: BTreeMap<String, Vec<Option<u64>>>,
-    gauges: BTreeMap<String, Vec<Option<i64>>>,
-    histograms: BTreeMap<String, Vec<Option<HistogramSnapshot>>>,
+    counters: BTreeMap<String, Option<HashMap<String, String>>>,
```

Comment:
Since we know we want to have a `{ 'metric_type': 'counter/gauge/histogram'}` in each of these, I wonder if it would be nicer to just initialise them with that, rather than having them optional. That would also clean up the merge code in `finalize` where we could just take the metadata and insert it into the field without having to do any more processing then.

Reply:
We initialize with the metric metadata and insert the type later.